frontend/src/bundles/common/components/file-upload/file-upload.tsx

```diff
@@ -6,7 +6,7 @@ import { Button, type ButtonProperties } from '../components.js';
 type Properties = {
     accept: string;
     buttonProps: Partial<ButtonProperties>;
-    onChange: (event: React.ChangeEvent<HTMLInputElement>) => boolean;
+    onChange: (event: React.ChangeEvent<HTMLInputElement>) => unknown;
```

Comment:
why `unknown`?

Reply:
because I use some different logic from Veronica and my handleUploadFile doesn't return boolean value